src/main/resources/application.properties

```diff
@@ -13,7 +13,9 @@ spring.jpa.hibernate.ddl-auto=update
 
 spring.datasource.driver-class-name=org.postgresql.Driver
 
-# Set Cache SIZE
+# Set Cache SIZE and Limit SIZE
```

Comment:
limit on what? explain to the use what he is limiting. think that he is a monkey he don't understand what you did and how you wrote the code you need to be esamption free. we can't asume that the user knows or understand something, he is an idiot.

Reply:
fixed